rules/windows/process_creation/win_susp_use_of_te_bin.yml

```diff
@@ -0,0 +1,27 @@
+title: Malicious WSC (Windows Script Components) File Execution by TAEF Detection
+id: 634b00d5-ccc3-4a06-ae3b-0ec8444dd51b 
+status: experimental
+description: Windows Test Authoring and Execution Framework (TAEF) framework allows you to run automation by executing tests files written on different languages (C, C#, Microsoft COM Scripting interfaces). Adversaries may execute malicious code (such as WSC file with VBScript, dll and so on) directly by running te.exe
+references:
+  - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OtherMSBinaries/Te.yml 
+  - https://twitter.com/pabraeken/status/993298228840992768 
+  - https://docs.microsoft.com/en-us/windows-hardware/drivers/taef/
+tags:
+    - attack.t1218
+author: Agro (@agro_sev) oscd.community
```

Comment:
I missed one moment: you need to put the `author` field to the quotes, as there is a `@` character.

Reply:
Thank you very much, @uncleAntik!